### ISSUE TITLE
about cldice loss

### ISSUE THREAD

**Issue by long123524** [jocpae/clDice#9]
Thank you very much for your article. What I want to ask is whether cldice is used as a module? If I have multiple tasks, how should I combine multiple loss functions for training?

**Comment by jocpae** [comment]
Hi, I will answer this very similar to your previous issue: 
As described in detail in our paper you can combine clDice with many other loss functions, e.g. BCE, MAE or Dice. We usually use simple linear weighting for this. This is also implemented in the `soft_dice_cldice` class in our code. For details and experiments how the alpha parameter influences the training please see our paper (and supplementary material). If you are referring to a Mutli-class segmentation problem, I have to say that unfortunately clDice cannot handle multi-class segmentation at this point.

**Comment by long123524** [comment]
Thank you for your reply. What I want to ask is whether cldice is also applicable to other segmentation tasks? For example, building extraction. In addition, can cldice be used for multi-task neural networks?

**Comment by jocpae** [comment]
Indeed clDice can be used for any binary segmentation task. The weighting with other loss functions is a problem specific parameter though. Unfortunately we have not worked with clDice for multi-task learning.

**Comment by long123524** [comment]
When I call this class, I always get the following error. 
![image](https://user-images.githubusercontent.com/73996375/127417768-7808a8e8-5bc8-49f9-813c-94135f3dfa46.png)

I am confused, how should the clide class call it? My calling method is as shown below
![image](https://user-images.githubusercontent.com/73996375/127417929-fadef3bf-b737-43c9-90e1-89116da0931c.png)
My task is a binary classification task

**Comment by long123524** [comment]
My task is a binary classification task，My calling cldice is as shown below
![image](https://user-images.githubusercontent.com/73996375/127440887-8d9674f9-7e47-4313-a89f-07ef1ee7381d.png)
but, Some errors occurred，as shown in this figure, How should I solve this error？
![image](https://user-images.githubusercontent.com/73996375/127441146-df6cdd3d-54d7-4928-a2ad-6cd30f5e83d4.png)
